Anlab.Mvc/Views/Shared/Layout.cshtml

```diff
@@ -160,5 +160,13 @@
     <script src="~/dist/root.js" asp-append-version="true"></script>
 
     @RenderSection("Scripts", required: false)
+    <script async src=“https://www.googletagmanager.com/gtag/js?id =UA-652647-50“></script>
```

Comment:
@jSylvestre also it looks like you have a space in the URL that i don't think should be there